### ISSUE TITLE
About Deployment

### ISSUE THREAD

**Issue by sahinyanlik** [iliakan/javascript-tutorial-server#12]
Hello @iliakan 

,

Thanks for this server. I did exactly what you said and except one problem I was able to install i think there is a problem with bemto.pug and node v8.11.4. After I fixed the issue I was able to run the project by ./edit tr and I think it worked. I said I think because there is no signal about it. You can see it in attachment 

this is my console, as you see I give permission to 3000 port.

![gcloud2](https://user-images.githubusercontent.com/975479/44228106-1b9cd300-a19d-11e8-92ab-49e9cd3afd8d.png)

And this is what it seems like at the end of ./edit tr 

![gcloud](https://user-images.githubusercontent.com/975479/44228107-1c356980-a19d-11e8-8590-d734fac3d042.png)

I thought I should be able to reach this code by http://my-ip:3000

Is there anything that I need to do ?

Thanks.

**Comment by iliakan** [comment]
Yeah, should work. I don't see the full log, only import ending (that looks fine).

You might want to run it locally, like other people do.
Maybe it'll just work w/o messing with debug.
